pyproject.toml

```diff
@@ -35,6 +35,7 @@ cryptography = "^41.0.4"
 dateparser = "^1.1.8"
 mail-parser = "^3.15.0"
 requests = "^2"
+pre-commit = "^3.5.0"
```

Comment:
@andrewfrench move this to the `dev` [group](https://python-poetry.org/docs/master/managing-dependencies/). And we should probably add a bit to the contributing guidelines that says they'll need to install dev dependencies via `poetry install --with dev`.